### PR TITLE
Use singular form for serializer names.

### PR DESCRIPTION
**Pull request by RayeN**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # [Changelog](https://github.com/yola/drf-madprops)
 
+## 0.1.0
```
Comment (jouberthenk) [inline CHANGELOG.md]: if we're using semantic versioning then this is an api breaking change so you bump the major version

Comment (RayeN) [inline CHANGELOG.md]: we are less than 1.x.x - as result bumping minor version is fine.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by stefanor** [comment]
That does look better :+1:

**Pull request merged**